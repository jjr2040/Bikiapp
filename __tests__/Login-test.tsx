import 'react-native';
import React from 'react'
import renderer from 'react-test-renderer'
import { LoginForm } from '../app/pages/user/LoginForm.page';

jest.useFakeTimers();

test('renders correctly', () => {
  const tree = renderer.create(<LoginForm />).toJSON();
  expect(tree).toMatchSnapshot();
});