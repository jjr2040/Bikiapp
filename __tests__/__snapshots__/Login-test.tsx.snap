// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        undefined,
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableOnAndroid={false}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        },
        Object {
          "paddingLeft": 10,
          "paddingRight": 10,
        },
      ]
    }
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <View
        style={Object {}}
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "marginTop": 15,
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "left": 0,
                "opacity": 1,
                "paddingBottom": undefined,
                "paddingTop": undefined,
                "position": "absolute",
                "right": 0,
                "top": 18,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#575757",
                  "fontSize": 17,
                  "left": 0,
                  "paddingRight": 5,
                  "paddingTop": 5,
                  "top": 6,
                }
              }
            >
              <Text
                style={
                  Object {
                    "fontSize": 17,
                  }
                }
              >
                Email
              </Text>
            </Text>
          </View>
          <TextInput
            allowFontScaling={true}
            autoCapitalize="none"
            editable={true}
            keyboardType="email-address"
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            placeholderTextColor="#575757"
            rejectResponderTermination={true}
            style={
              Object {
                "color": "#000",
                "flex": 1,
                "fontSize": 17,
                "height": 50,
                "marginLeft": null,
                "paddingBottom": 7,
                "paddingLeft": null,
                "paddingRight": 5,
                "paddingTop": 3,
                "top": 10,
              }
            }
            underlineColorAndroid="rgba(0,0,0,0)"
            value=""
          />
        </View>
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 1,
              "borderColor": "#D9D5DC",
              "borderLeftWidth": 0,
              "borderRadius": undefined,
              "borderRightWidth": 0,
              "borderTopWidth": 0,
              "borderWidth": undefined,
              "flexDirection": "row",
              "marginLeft": 15,
              "marginTop": 15,
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "left": 0,
                "opacity": 1,
                "paddingBottom": undefined,
                "paddingTop": undefined,
                "position": "absolute",
                "right": 0,
                "top": 18,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#575757",
                  "fontSize": 17,
                  "left": 0,
                  "paddingRight": 5,
                  "paddingTop": 5,
                  "top": 6,
                }
              }
            >
              <Text
                style={
                  Object {
                    "fontSize": 17,
                  }
                }
              >
                Contraseña
              </Text>
            </Text>
          </View>
          <TextInput
            allowFontScaling={true}
            editable={true}
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            placeholderTextColor="#575757"
            rejectResponderTermination={true}
            secureTextEntry={true}
            style={
              Object {
                "color": "#000",
                "flex": 1,
                "fontSize": 17,
                "height": 50,
                "marginLeft": null,
                "paddingBottom": 7,
                "paddingLeft": null,
                "paddingRight": 5,
                "paddingTop": 3,
                "top": 10,
              }
            }
            underlineColorAndroid="rgba(0,0,0,0)"
            value=""
          />
        </View>
      </View>
    </View>
  </RCTScrollView>
  <View
    style={
      Object {
        "backgroundColor": "#F8F8F8",
        "borderColor": "#cbcbcb",
        "borderTopWidth": 0.5,
        "elevation": 3,
        "flexDirection": "row",
        "height": 55,
        "justifyContent": "center",
        "left": 0,
        "paddingBottom": 0,
        "right": 0,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "stretch",
          "backgroundColor": "#007aff",
          "borderBottomWidth": null,
          "borderColor": "#007aff",
          "borderLeftWidth": null,
          "borderRadius": 0,
          "borderRightWidth": null,
          "borderTopWidth": null,
          "borderWidth": undefined,
          "elevation": 2,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 6,
          "paddingTop": 6,
          "shadowColor": undefined,
          "shadowOffset": undefined,
          "shadowOpacity": undefined,
          "shadowRadius": undefined,
        }
      }
    >
      <Text
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "#fff",
            "fontFamily": "System",
            "fontSize": 16.5,
            "marginLeft": 0,
            "marginRight": 0,
            "paddingLeft": 16,
            "paddingRight": 16,
          }
        }
        uppercase={false}
      >
        Iniciar sesión
      </Text>
    </View>
  </View>
</View>
`;
